src/main/java/hudson/plugins/jira/CredentialsHelper.java

```diff
@@ -0,0 +1,84 @@
+package hudson.plugins.jira;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.annotation.CheckForNull;
+
+import com.cloudbees.plugins.credentials.CredentialsMatchers;
+import com.cloudbees.plugins.credentials.CredentialsProvider;
+import com.cloudbees.plugins.credentials.CredentialsScope;
+import com.cloudbees.plugins.credentials.SystemCredentialsProvider;
+import com.cloudbees.plugins.credentials.common.StandardUsernamePasswordCredentials;
+import com.cloudbees.plugins.credentials.domains.URIRequirementBuilder;
+import com.cloudbees.plugins.credentials.impl.UsernamePasswordCredentialsImpl;
+import hudson.security.ACL;
+import hudson.util.Secret;
+import jenkins.model.Jenkins;
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * Helper class for vary credentials operations.
+ *
+ * @author Zhenlei Huang
+ */
+public class CredentialsHelper {
```

Comment:
please add tests for this class

Reply:
OK, I'll do it :)

Reply:
Tests for CredentialsHelper added. Commit: https://github.com/jenkinsci/jira-plugin/pull/140/commits/0966980a0efd13835bcbc0a1f6337d0ee1c1c69c